backend/src/main/java/com/team4/backend/exception/InvalidPageRequestException.java

```diff
@@ -1,5 +1,8 @@
 package com.team4.backend.exception;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
```

Comment:
I see the ResponseStatus import but no annotations and @NoArgsConstructor is missing